tests/Auth0.ManagementApi.IntegrationTests/RolesTests.cs

```diff
@@ -217,8 +218,8 @@ public async Task Test_roles_assign_unassign_permission_to_role()
             role.Description.Should().Be(newRoleRequest.Description);
 
             // Get a resource server
-            var resourceServer = await _apiClient.ResourceServers.GetAsync("5cccc711773967081270a036");
-            var originalScopes = resourceServer.Scopes.ToList();
+            var resourceServer = await _apiClient.ResourceServers.GetAsync("dotnet-testing");
+            var originalScopes = resourceServer.Scopes != null ? resourceServer.Scopes.ToList() : new List<ResourceServerScope>();
```

Comment:
On the old tenant, the API being queried had scopes defined. In the new tenant this wasnt the case.
Instead of configuring the tenant, I updated the tests so that it can work with and without existing scopes.